scripts/scheming_migration/index.js

```diff
@@ -315,31 +341,31 @@ async function main() {
 				packageExtras['edc_state'] = 'DRAFT';
 			}
 			if (!('resource_status' in packageExtras)) {
-				packageExtras['resource_status'] = 'underDevelopment';
+				packageExtras['resource_status'] = 'onGoing';
 				await pool.query("INSERT INTO package_extra (id, package_id, key, value, revision_id, state)" +
 									"VALUES ($4, $1, 'resource_status', $2, $3, 'active')", 
 									[packageObj['id'], packageExtras['resource_status'], packageObj['revision_id'], uuidv4()]);
 			}
 			if (packageExtras['resource_status'] === 'obsolete' && !('replacement_record' in packageExtras)) {
-				packageExtras['replacement_record'] = 'Not Set. This needs to be updated';
+				packageExtras['replacement_record'] = null;
 				await pool.query("INSERT INTO package_extra (id, package_id, key, value, revision_id, state)" +
 									"VALUES ($4, $1, 'replacement_record', $2, $3, 'active')", 
 									[packageObj['id'], packageExtras['replacement_record'], packageObj['revision_id'], uuidv4()]);
```

Comment:
The 'dates' field has two primary options, Published and Created and is a mandatory field. Almost all other resources in the DB have ether a Published or a Created date associated with them (there may be specific cases where they dont, but those would be in error and the script is not meant to handle them). But 'webservice' and 'application' never had a Published or Created date associated with them in this format and since we are making it a mandatory field, instead of breaking all the 'webservice' and 'application' packages, it was decided to add a Created date, based on their record_create_date.

Reply:
While the reply is for another conversation this does appear to have been resolved - thank you!